src/services/post.ts

```
@@ -0,0 +1,25 @@
+export const PostGQL = {
+  MUTATIONS: {
+    CREATE_POST: gql`
+      mutation CreatePost($postCreateInput: PostCreateInput!) {
+        createPost(postCreateInput: $postCreateInput)
+      }
+    `,
+  },
+};
+
+export const createPost = async (
+  client: ApolloClient<any>, 
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)